packages/apps-config/src/api/spec/subspace.ts

```diff
@@ -39,14 +35,11 @@ function extractAuthor (
   api: ApiInterfaceRx
 ): FarmerPublicKey | undefined {
   const preRuntimes = digest.logs.filter(
-    ({ isPreRuntime, type }) => isPreRuntime && type.toString() === 'SUB_'
-  );
-  const { solution }: SubPreDigest = api.registry.createType(
-    'SubPreDigest',
-    preRuntimes[0]
+    (log) => log.isPreRuntime && log.asPreRuntime[0].toString() === 'SUB_'
   );
+  const { solution }: SubPreDigest = api.registry.createType('SubPreDigest', preRuntimes[0].asPreRuntime[1]);
```

Comment:
What does `[1]` do in `.asPreRuntime[1]`? It is not obvious to me

Reply:
[0] is the "_SUB" consensus engine Id, and [1] is the data as bytes, in our case, it contains the "SubPreDigest" where we can get the Solution and the farmer public key.

Reply:
Ah, sorry, yes, it makes sense now, thanks!